concent_api/core/decorators.py

```diff
@@ -316,3 +319,20 @@ def wrapper(request: HttpRequest, golem_message: message.Message, client_public_
         response_from_view = view(request,  golem_message, client_public_key)
         return response_from_view
     return wrapper
+
+
+def retry_middleman_connection_if_not_pass_timeout(send_request_to_middleman: Callable) -> Callable:
+
+    @wraps(send_request_to_middleman)
+    def wrapper(middleman_message: GolemMessageFrame) -> bytes:
```

Comment:
Please provide the mentioned description on why implementation differs from description.